doc_classes/TrainController.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TrainController" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/godotengine/godot/master/doc/class.xsd">
+	<brief_description>
+		Base train controller controlling all train systems
+	</brief_description>
+	<description>
+		Base train controller
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_brake" qualifiers="const">
+			<return type="TrainBrake" />
+			<description>
+				Returns TrainBrake class instance
+			</description>
+		</method>
+		<method name="get_engine" qualifiers="const">
+			<return type="TrainEngine" />
+			<description>
+				Returns TrainEngine class instance
+			</description>
+		</method>
+		<method name="get_mover_state">
+			<return type="Dictionary" />
+			<description>
+				Returns dictionary of values from iternal mover. Useful for debugging
+			</description>
+		</method>
+		<method name="get_security_system" qualifiers="const">
+			<return type="TrainSecuritySystem" />
+			<description>
+				Returns TrainSecuritySystem class
+			</description>
+		</method>
+		<method name="update_mover" qualifiers="const">
+			<return type="void" />
+			<description>
+				Updates iternal mover
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="dimensions/mass" type="float" setter="set_mass" getter="get_mass" default="0.0">
+			Mass of the train
+		</member>
+		<member name="nominal_battery_voltage" type="float" setter="set_nominal_battery_voltage" getter="get_nominal_battery_voltage" default="0.0">
+			Nominal voltage of the train's battery
+		</member>
+		<member name="parts/brake" type="NodePath" setter="set_brake_path" getter="get_brake_path" default="NodePath(&quot;&quot;)">
+			Path to the TrainBrake node
+		</member>
+		<member name="parts/engine" type="NodePath" setter="set_engine_path" getter="get_engine_path" default="NodePath(&quot;&quot;)">
+			Path to the TrainEngine node
+		</member>
+		<member name="parts/security_system" type="NodePath" setter="set_security_system_path" getter="get_security_system_path" default="NodePath(&quot;&quot;)">
+			Path to the TrainSecuritySystem node
+		</member>
+		<member name="switches/battery_enabled" type="bool" setter="set_battery_enabled" getter="get_battery_enabled" default="false">
+			Is battery enabled
```

Comment:
Toggle train battery

Reply:
Tak można funkcje nazwać że toggluje train Battery. To jest opis wartości więc sugerował bym opisanie że ona wskazuje czy bateria włączona czy nie

Reply:
Racja. To może "Main battery state" lub coś takiego? Chodzi mi bardziej o to, że "is cośtam enabled" sugeruje readonly, a to property którym również zmieniamy stan. Oczywiście to jest niuans i może moje skrzywione spojrzenie. Zostawiam Twojej decyzji.